CMakeLists.txt

```diff
@@ -210,6 +210,7 @@ string(TOUPPER "$ENV{ARCH_NAME}" ARCH_NAME_DEF)
 add_compile_definitions(ARCH_${ARCH_NAME_DEF})
 add_compile_options(
     -Werror
+    -Wno-deprecated-declarations
```

Comment:
Can you please check, there are compiler specific flags below
like this for clang    "$<$<CXX_COMPILER_ID:Clang>:-Wno-deprecated-declarations>"

Reply:
removed the compiler specific